### PR TITLE
Fixed some bugs with config file

### PR DESCRIPTION
**Pull request by madnx**
- Fixed confide::name to confide.name in 'config', 'mail', 'view'.
- Moved test/confide.php to config/
- Moved tests/Confide/helpers.php to src/Confide since may not exist in deployed sites

### CODE REVIEW

File: src/Confide/CacheLoginThrottleService.php [hunk 1]
```diff
@@ -15,14 +15,14 @@ class CacheLoginThrottleService implements LoginThrottleServiceInterface
     /**
      * Laravel application.
      *
-     * @var \Illuminate\Contracts\Foundation\Application
+     * @var \Illuminate\Foundation\Application
```
Comment (GrahamCampbell) [inline src/Confide/CacheLoginThrottleService.php]: no

File: src/Confide/CacheLoginThrottleService.php [hunk 2]
```diff
@@ -15,14 +15,14 @@ class CacheLoginThrottleService implements LoginThrottleServiceInterface
...
      */
     public $app;
 
     /**
      * Create a new PasswordService.
      *
-     * @param \Illuminate\Contracts\Foundation\Application $app Laravel application object.
```
Comment (GrahamCampbell) [inline src/Confide/CacheLoginThrottleService.php:25]: wrong

File: src/Confide/Confide.php [hunk 3]
```diff
@@ -46,7 +46,7 @@ class Confide
      * @param \Zizaco\Confide\RepositoryInterface           $repo
      * @param \Zizaco\Confide\PasswordServiceInterface      $passService
      * @param \Zizaco\Confide\LoginThrottleServiceInterface $loginThrottler
-     * @param \Illuminate\Contracts\Foundation\Application            $app            Laravel application object
```
Comment (GrahamCampbell) [inline src/Confide/Confide.php:49]: all these changes are wrong

Comment (madnx) [inline src/Confide/Confide.php:49]: Hum that's not from me. The code is the one from the source Zizaco. Going to make a commit about that in a sec!

EDIT : Yep sorry I didn't read well. Going to commit though.

**Comment by madnx** [comment]
Fixed some bugs that occured while merging two commits.

**Comment by madnx** [comment]
Ended up create a new pull-request. I had a lot of bugs with my PHPStorm which forced me to reinstall it. Sorry about that guys!

**Pull request closed without merging**